tests/system/test_aroi_v3_live_smoke.py

```
@@ -0,0 +1,119 @@
+        headers=ua, timeout=30,
+    ).json()
+    onionoo_data = requests.get(
+        "https://onionoo.torproject.org/details"
+        "?fields=fingerprint,nickname,contact,country,first_seen",
+        timeout=60,
+    ).json()
+
+    from allium.lib.relays import Relays
+    from allium.lib.aroi_validation import get_contact_validation_status
+
+    with patch.object(Relays, "__init__", lambda x, **kwargs: None):
```

Ruff (0.15.12)

[warning] Unused lambda argument: `x`

(ARG005)